tests/acceptance/features/bootstrap/NotificationContext.php

```diff
@@ -315,10 +315,26 @@ public function filterResponseByNotificationSubjectAndResource(string $subject,
 	 * @param TableNode $table
 	 *
 	 * @return void
+	 * @throws Exception
 	 */
 	public function userShouldGetANotificationWithMessage(string $user, string $subject, TableNode $table):void {
-		$this->userListAllNotifications($user);
-		$actualMessage = str_replace(["\r", "\n"], " ", $this->filterResponseAccordingToNotificationSubject($subject)->message);
+		$count = 0;
+		// sometimes the test might try to get notification before the notification is created by the server
+		// in order to prevent test from failing because of that try to list the notifications again
+		do {
+			if ($count > 0) {
+				\sleep(1);
+			}
+			$this->featureContext->setResponse(null);
+			$this->userListAllNotifications($user);
+			$this->featureContext->theHTTPStatusCodeShouldBe(200);
+			++$count;
```

Comment:
though it doesn't have any effect on current implementation :sweat_smile: 
```suggestion
			$count++;
```

Reply:
I think it's okay

Reply:
It's okey then. It was just my eyes :smile: